### ISSUE TITLE
多分类

### ISSUE THREAD

**Issue by 1125690278** [BitVoyage/FastBERT#10]
我多分类的时候用验证的脚本加载蒸馏好的模型结果有56%，speed为0时的结果只有37%，不应该是一样的吗？

**Comment by 1125690278** [comment]
而且我speed从0.1变到0.8 acc没有变化

**Comment by ZhaoJiangJie** [comment]
I met the same issues as yours, and i print the value uncertain,it's a quite nearly 1.0,so it will pass to the last layer of bert,so the output always generate from the last layer.and I found when execute the stage_1 of self distillation, the uncertain  has nearly 1.0  too.
![image](https://user-images.githubusercontent.com/17317538/98214972-24e30780-1f82-11eb-872f-0edc633ed6e9.png)
stage1 uncertain
![image](https://user-images.githubusercontent.com/17317538/98215735-1cd79780-1f83-11eb-87f3-5b10806686c7.png)
And I change the teacher logits code,then it seems that the value of uncertain began degrade,but the performance of inference accuracy degrade too, I'm try to find the reason now.
![image](https://user-images.githubusercontent.com/17317538/98215992-7213a900-1f83-11eb-9f82-760038b56881.png)

**Comment by ChristopheZhao** [comment]
I met the same issues as yours, and i print the value of uncertain,it's a quite nearly 1.0,so it will pass to the last layer of bert,so the output always generate from the last layer.and I found when execute the stage_1 of self distillation, the uncertain has nearly 1.0 too.
![image](https://user-images.githubusercontent.com/17317538/98216676-54930f00-1f84-11eb-9835-2a9b9615eebf.png)

And I change the teacher logits code,then it seems that the value of uncertain began degrade,but the performance of inference accuracy degrade too, I'm try to find the reason now.
![image](https://user-images.githubusercontent.com/17317538/98216715-62e12b00-1f84-11eb-8ae4-0bd1a128c830.png)

**Comment by BitVoyage** [comment]
分类数目很多时，FastBert加速效果不明显，已收到多个反馈

**Comment by 1125690278** [comment]
> 分类数目很多时，FastBert加速效果不明显，已收到多个反馈

但是我speed=0的时候不就等于不蒸馏吗，为啥验证脚本跑出来的结果和speed =0时会不一样，相差了20%

**Comment by BitVoyage** [comment]
请用最新的代码试试，有一个BUG修复
